src/frequenz/sdk/actor/power_distributing/_battery_status.py

```diff
@@ -197,14 +197,15 @@ def __init__(  # pylint: disable=too-many-arguments
             raise RuntimeError(f"Can't find inverter adjacent to battery: {battery_id}")
 
         self._battery: _ComponentStreamStatus = _ComponentStreamStatus(
-            battery_id, data_recv_timer=Timer(max_data_age_sec)
+            battery_id,
+            data_recv_timer=Timer.timeout(timedelta(max_data_age_sec)),
         )
         self._inverter: _ComponentStreamStatus = _ComponentStreamStatus(
-            inverter_id, data_recv_timer=Timer(max_data_age_sec)
+            inverter_id,
+            data_recv_timer=Timer.timeout(timedelta(max_data_age_sec)),
```

Comment:
Took a while to identify why the timers weren't triggering.  These have to say `timedelta(seconds=max_data_age_sec)`, else it assumes days.  

@llucax next pylint project?

Reply:
I think it should be part of `pylint-datetime`.

Reply:
Yes, it will be

Reply:
@jh2007github could you please update how the timedelta objects are constructed here,  and let us know when the PR is ready for review again?

Reply:
have you pushed?  I don't see it yet.

Reply:
oh it is done in a different commit.  I guess that's fine.